src/ignitus-UserInterfaceKit/Components/Atoms/interfaceInputs/Components/interfaceInputs.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { InputContainer } from '../styles';
+import { StyledHeading2 } from '../../../../styles';
+import { InputField1 } from '../../../../../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/input';
+
+const interfaceInputs = () => (
+  <InputContainer>
+  <StyledHeading2>Input Fields ★</StyledHeading2>
+  <hr />
+    <InputField1 placeholder="First Name" type="text" />
```

Comment:
And please create a component for an input that will receive the `props`  for example have a look here - https://github.com/Ignitus/Ignitus-client/blob/develop/src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultCheckbox/Components/defaultCheckbox.tsx

You can't hardcode type="text" so `type` here will be property that can receive a `string` that can string can be `password`, `text` etc, and `placeholder`, `name` will also be a property.

`<input type={type} placeholder={placeholder} name={name}>`


For example see these 2 use cases and how they are being used.

https://github.com/Ignitus/Ignitus-client/blob/develop/src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultCheckbox/Components/defaultCheckbox.tsx

https://github.com/Ignitus/Ignitus-client/blob/develop/src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultDropdown/Components/defaultDropdown.tsx